code/modules/mob/living/simple_animal/hostile/wendigo.dm

```diff
@@ -0,0 +1,236 @@
+
+				if(ishuman(mob_target))
+					if(mob_target.health < -400)
+						visible_message("<span class = 'warning'>\The [src] is trying to eat \The [mob_target]!</span>","<span class = 'warning'>You hear crunching.</span>")
+						if(do_after(src, mob_target, 50, needhand = FALSE))
+							consumes += 1
+							names += mob_target.real_name
+							qdel(mob_target)
+
+			return
+	. =..()
+
+/mob/living/simple_animal/hostile/wendigo/Life()
+	/*Check evolve like zombies, run away from fire,
+	mimic speech if anyone's nearby but just out of eyesight
+	*/
+	..()
+	if(!isDead())
+		check_evolve()
+
+		var/list/can_see = view(src, vision_range)
+
+		for(var/obj/machinery/space_heater/campfire/fire in can_see)
+			var/dist = get_dist(src, fire)
+			if(dist < (fire.light_range*2))
+				walk_away(src,fire,(fire.light_range*2),move_to_delay)
+
+
+		if(names.len)
+			speak_chance = 15
+
+
+/mob/living/simple_animal/hostile/wendigo/GetVoice()
+	if(names.len)
+		return pick(names)
+	else
+		speak_chance = 0
+		return name
+/mob/living/simple_animal/hostile/wendigo/proc/check_evolve()
+	return
+
+/mob/living/simple_animal/hostile/wendigo/human
+	icon_state = "wendigo"
+	icon_living = "wendigo"
+	icon_dead = "wendigo_dead"
+
+	speed = 1
+
+/mob/living/simple_animal/hostile/wendigo/human/death()
+	flick("wendigo_dying",src)
+	..()
+
+/mob/living/simple_animal/hostile/wendigo/human/check_evolve()
+	if(consumes > HUMEVOLV)
+		var/mob/living/simple_animal/hostile/wendigo/evolved/new_wendigo = new /mob/living/simple_animal/hostile/wendigo/evolved(src.loc)
+		new_wendigo.names = names
+		qdel(src)
+
+/mob/living/simple_animal/hostile/wendigo/evolved
+	icon = 'icons/mob/48x48.dmi'
+	icon_state = "wendigo_med"
+	icon_living = "wendigo_med"
+	icon_dead = "wendigo_med_dead"
+	pixel_x = -8 * PIXEL_MULTIPLIER
+	health = 250
+	maxHealth = 250
+
+	melee_damage_lower = 25
+	melee_damage_upper = 45
+
+/mob/living/simple_animal/hostile/wendigo/evolved/check_evolve()
+	if(consumes > EVOLEVOLV)
+		var/number_of_alpha
+		for(var/mob/living/simple_animal/hostile/wendigo/alpha/A in wendigo_alphas)
+			var/datum/zLevel/L = get_z_level(A)
+			if(istype(L,/datum/zLevel/centcomm))
+				continue //Damn it admins
+			if(A.isDead())
+				continue
+			number_of_alpha += 1
+			break
+		if(!number_of_alpha)
+			var/mob/living/simple_animal/hostile/wendigo/alpha/new_wendigo = new /mob/living/simple_animal/hostile/wendigo/alpha(src.loc)
+			new_wendigo.names = names
+			qdel(src)
+
+var/list/wendigo_alphas()
+
+/mob/living/simple_animal/hostile/wendigo/alpha
+	desc = "You can't help but feel that, no matter what, you should have brought a bigger gun."
+	icon = 'icons/mob/giantmobs.dmi'
+	icon_state = "wendigo_noblood"
+	icon_living = "wendigo_noblood"
+	icon_dead = "wendigo_dead"
+	var/icon_enraged = "wendigo"
+	pixel_x = -16 * PIXEL_MULTIPLIER
+	health = 600
+	maxHealth = 600
+	speed = 5
+	melee_damage_lower = 35
+	melee_damage_upper = 50 //Jesus christ
```

Comment:
And it is an entirely unique mob, of which there can't be two of at a time unless through bus means.